ansible/roles/uan_disk_config/README.md

```diff
@@ -33,38 +33,76 @@ uan_require_disk: false
 
 ### `uan_device_name_filter`
 
-Regular expression of disk device name for this role to filter.
+Regular expression of disk device names for this role to filter.
 Input to the `device_filter` module.
 
 ```yaml
 uan_device_name_filter: "^sd[a-f]$"
 ```
 
+### `uan_device_name_exclude_filter`
+
+Regular expression of disk device names to exclude for this role.
+Input to the `device_filter` module.
+
+```yaml
+uan_device_name_exclude_filter: ""
+```
+
 ### `uan_device_host_filter`
 
-Regular expression of host for this role to filter.
+List of hosts for this role to filter.
+Input to the `device_filter` module.
+
+```yaml
+uan_device_host_filter: []
+```
+
+### `uan_device_host_exclude_filter`
+
+List of hosts to exclude for this role to filter.
 Input to the `device_filter` module.
 
 ```yaml
-uan_device_host_filter: ""
+uan_device_host_filter: []
 ```
 
 ### `uan_device_model_filter`
 
-Regular expression of device model for this role to filter.
+List of device models for this role to filter.
 Input to the `device_filter` module.
 
 ```yaml
-uan_device_model_filter: ""
+uan_device_model_filter: []
+```
+
+### `uan_device_model_exclude_filter`
+
+List of device models to exclude for this role.
+Input to the `device_filter` module.
+
+```yaml
+uan_device_model_exclude_filter: []
 ```
 
 ### `uan_device_vendor_filter`
 
-Regular expression of disk vendor for this role to filter.
+List of disk vendors for this role to filter.
+Input to the `device_filter` module.
+
+```yaml
+uan_device_vendor_filter: []
+```
+
+### `uan_device_vendor_exclude_filter`
+
+List of disk vendors to exclude for this role.
+Default is `["LIO-ORG"]` and  must be excluded
+if SBPS is used for the image projection service.
```

Comment:
If this is the right definition, suggested change to define it for those unaware:
 
```suggestion
if iSCSI Based Projection Service, SBPS, is used for
the image projection service.
```